content/03.methods.md

```diff
@@ -106,10 +106,18 @@ Per recommendations by the New York Genome Center [@doi:10.1101/623702], the Lan
 
 We filtered outputs from both callers on the "PASS" filter, and annotated using The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, and created MAFs using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.17.
 
+#### Consensus SNV Calling
+
+Mutations were considered to be the same across callers if they were identical in the following MAF fields: `Chromosome`, `Start_Position`, `Reference_Allele`,  `Allele`, and `Tumor_Sample_Barcode`.
+All four callers were compared in their overlap with each other, variant allele frequencies of their calls, base changes called, and genomic regions of their calls [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/compare_snv_callers_plots.Rmd].
```

Comment:
```suggestion
We compared all four callers in their overlap with each other, variant allele frequencies of their calls, base changes called, and genomic regions of their calls [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/compare_snv_callers_plots.Rmd].
VarDict called numerous variants with low allele frequency (Figure? Supplementary Figure? Link to GitHub?) in addition to nearly all of the calls identified by the other three callers.
Because our filtered set was based on the intersection, we removed it from the consensus calculation.
```
Did _everything_ called by the other three callers also get called by vardict? I.e., how much does this filtering affect your results?